torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -2815,36 +2877,53 @@ bool insertProfileIValue(ProfilingRecord* pr, Node* node, size_t offset) {
       // argument 1: reduction sizes;
       case 1:
         // TODO(profile_size): double check optional[size]?
-        profileSize(pr, node, offset);
+        profileReductionSize(pr, node, offset);
         break;
       default:
         return false;
     }
     return true;
   }
 
-  /*
-  // TODO: Enable view in parser by detecting non-alias view operation
   static auto view_schema =
       getOperatorForLiteral(
           "aten::view(Tensor(a) self, int[] size) -> Tensor(a)")
           ->schema();
-  static auto reshape_schema =
+  static auto view_copy_schema =
       getOperatorForLiteral(
-          "aten::reshape(Tensor(a) self, int[] shape) -> Tensor(a)")
+          "prim::view_copy(Tensor(a) self, int[] size) -> Tensor(a)")
           ->schema();
-  if (node->matches(view_schema) || node->matches(reshape_schema)) {
+  if (node->matches(view_schema) || node->matches(view_copy_schema)) {
     switch (offset) {
       // argument 1: new tensor size;
       case 1:
-        profileSize(pr, node, offset);
+        profileIntList(pr, node, offset);
```

Comment:
I'd suggest create a new profileViewSize. So we won't have the aten::eq check at runtime.